src/scss/grommet-core/_objects.search-input.scss

```diff
@@ -1,53 +1,54 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
+.grommet .search-input,
 .search-input {
   position: relative;
   display: inline-block;
-}
 
-.search-input__input {
-  width: 100%;
-  height: 100%;
-  display: block;
-  padding-right: round($inuit-base-spacing-unit * 2.5);
+  &__input {
```

Comment:
Can you re-flatten the SCSS? I've been slowly migrating this way based on some articles on SCSS structuring I've been reading.